drivers/spi/spi_mcux_flexcomm.c

```diff
@@ -218,6 +241,378 @@ static int spi_mcux_configure(const struct device *dev,
 	return 0;
 }
 
+#ifdef CONFIG_SPI_MCUX_FLEXCOMM_DMA
+
+/* dummy value used for transferring NOP when tx buf is null
+ * and use as dummy sink for when rx buf is null
+ */
+uint32_t dummy_tx_buffer;
+uint32_t dummy_rx_buffer;
+/* Used to send the last word */
+uint32_t last_word;
```

Comment:
This should be in `struct spi_mcux_data`, otherwise I don't think this driver will work with multiple instances.

Reply:
Thanks for catching this, I will make this change